src/simulator/src/data/project.ts

```diff
@@ -91,10 +91,12 @@ export function projectSavedSet(param) {
  * @category data
  */
 export async function saveOffline() {
-    const data = await generateSaveData()
+    const data = await generateSaveData('')
```

Comment:
Curious to know why passed `''`?

Reply:
because parameter of type string is required